src/Modal/Modal.js

```diff
@@ -34,27 +34,33 @@ class Modal extends Component {
     render() {
         const {
             actions,
+            backdropClassName,
             bodyProps,
             children,
-            className,
```

Comment:
Since this is a standard prop name in React, I would leave this and map it to either of the two divs (probably modal).